scripts/run-full-build-quiet.ps1

```
@@ -0,0 +1,251 @@
+
+$timestamp = Get-Date -Format "yyyyMMdd-HHmmss"
+$logFile = Join-Path $logDir "full-build-$timestamp.log"
+$stdoutFile = [System.IO.Path]::GetTempFileName()
+$stderrFile = [System.IO.Path]::GetTempFileName()
+
+try {
+    $timeoutSeconds = Positive-Int-OrDefault $env:QUIET_BUILD_TIMEOUT_SECONDS 180 0
+    $heartbeatSeconds = Positive-Int-OrDefault $env:QUIET_BUILD_HEARTBEAT_SECONDS 60 1
+
+    $mavenCommand = "mvn"
+    $isWindows = [System.Environment]::OSVersion.Platform -eq [System.PlatformID]::Win32NT
```

PSScriptAnalyzer (1.25.0)

[error] Starting from PowerShell 6.0, the Variable 'isWindows' cannot be assigned any more since it is a readonly automatic variable that is built into PowerShell, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)